_common/GFMAlerts.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
```

Comment:
We should add the standard Keyman header:

https://github.com/keymanapp/keyman/wiki/Keyman-Code-Style-Guide#file-header